components/contribution-flow/PaymentMethodList.tsx

```diff
@@ -293,6 +296,10 @@ export default function PaymentMethodList(props: PaymentMethodListProps) {
           {value.key === STRIPE_PAYMENT_ELEMENT_KEY && checked && (
             <Box my={3}>
               <PayWithStripeForm
+                bilingDetails={{
+                  name: LoggedInUser?.collective?.name,
+                  email: LoggedInUser?.email,
```

Comment:
Since we're attaching the payment method to the Collective, should we use the fromAccount name?